sourcegraph/sourcegraph
# App Setup: setup new setup settings page

Related design issue:
- #50729
- Figma design: https://www.figma.com/file/8UUuaIUPVJDzp8U9ImNvA7/Connect-Cody-and-polish-setup-experience?node-id=210-513&t=eX8TwV3BHeNjaDhy-0
---

## Context 
This PR is the first PR about the new setup settings page; it adds the page itself and main shell layout and reuses the existing setup wizard component step. It's in the WIP stage, but it can be merged since changes to this PR are app-specific. 

It doesn't work in this PR 
- Page Layout isn't final. Based on the conversation with @almeidapaulooliveira, the page will be changed/simplified if there is a need for more app settings except local repositories (remote repositories will be gone since it's not the app priority) 
-  Local repositories action buttons don't work (the form will be revamped in the following PR)
- "Connect to sourcegraph.com" isn't complete, it's in discussion at the moment (do we need to have an additional UI there, or button in the user menu will be enough) 

## Test plan
- Check that all page item links are working properly
- Check that the current setup wizard doesn't have any regressions
